commit 67e275a60b0aea6a3f2a95f0be22dc22eb1e0698
Author: Felix Yan <felixonmars@archlinux.org>
Date:   Mon Jan 23 22:10:28 2017 +0800

    Add support for Qt 5.8+
    
    qpa/qplatformintegrationfactory_p.h is now under QtGui, using
    gui-private instead of platformsupport-private fixes build with latest
    Qt.
    
    Change-Id: Icacabdec98e0b40fedacf19ce659ab987511dd3e

diff --git a/dwidget/dwidget.pro b/dwidget/dwidget.pro
index 12fc68d..0bb3449 100644
--- a/dwidget/dwidget.pro
+++ b/dwidget/dwidget.pro
@@ -5,8 +5,13 @@ TARGET = dtkwidget
 
 DEFINES += LIBDTKWIDGET_LIBRARY
 
-QT += multimedia multimediawidgets platformsupport-private
-greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
+QT += multimedia multimediawidgets
+greaterThan(QT_MAJOR_VERSION, 4) {
+  QT += widgets
+  # Qt >= 5.8
+  greaterThan(QT_MAJOR_VERSION, 5)|greaterThan(QT_MINOR_VERSION, 7): QT += gui-private
+  else: QT += platformsupport-private
+}
 
 unix{
     QT += x11extras dbus
